.pre-commit-config.yaml

```diff
@@ -1,5 +1,5 @@
 default_language_version:
-    python: python3.11
+    python: python3.10
```

Comment:
I'm inclined to remove this entirely. I don't think it matters much which version we use